src/components/ChatApp/MessageListItem/generateMessageBubbleUtil.js

```diff
@@ -377,21 +376,75 @@ export const generateMessageBubble = (
         }
         break;
       }
-      case 'anchor': {
-        const { link, text } = action;
-        listItems.push(
-          generateAnchorBubble(
-            actionType,
-            index,
-            text,
-            link,
-            message,
-            latestUserMsgID,
-            showFeedback,
-          ),
-        );
+      case 'anchor':
+        {
+          const { link, text } = action;
+          if (link.substring(0, 25) === 'https://www.openstreetmap') {
```

Comment:
Check the case where `map` was previously implemented and remove it from there. Also, write this logic into a function and just call that function from here.